### PR TITLE
Bazel Integration test binary

### PR DESCRIPTION
**Pull request by ryancouto**
Problem

Bazel/bzutil related integration testing was being done manually

Solution

Create a binary to run a simple integration test using bazel/bzutil related functionality

Result

bazel-integration will run as a part of our travis CI test suite

### CODE REVIEW

File: tests/testhelpers/utils.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	"os/exec"
+	"strings"
+)
+
+func InstallBinary(name string) {
+	cmd := exec.Command("go", "install", "./binaries/"+name)
+	cmd.Run()
+}
+
+func GetGopath() (gopath string, err error) {
```
Comment (dgdonut) [inline tests/testhelpers/utils.go]: Dup of common/common.go:GetFirstGopath

File: binaries/bazel-integration/main.go [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+	}
+	if err = assertEqual(*actionDigest, expectedActionDigest); err != nil {
+		testhelpers.KillAndExit1(clusterCmds, err)
+	}
+	// Execute
+	b, err = execute(gopath, bazel.DigestToStr(actionDigest))
+	if err != nil {
+		testhelpers.KillAndExit1(clusterCmds, err)
+
+		log.Error(err)
```
Comment (dgdonut) [inline binaries/bazel-integration/main.go]: Redundant?

**Comment by dgdonut** [comment]
Otherwise LGTM for a simple end-to-end.

File: binaries/bazel-integration/main.go (hunk 2 above)
Comment (ryancouto) [reply binaries/bazel-integration/main.go]: thanks, missed that

**Review by dgdonut — APPROVED** [review]

**Pull request merged**